documentation/src/main/asciidoc/topics/proc_using_protostuff_marshaller.adoc

```diff
@@ -1,16 +1,12 @@
-[id='protostuff_marshalling']
-= Using the Protostuff Marshaller
-[NOTE]
-====
-Protostuff marshaller is now deprecated and planned for removal. You should replace any Protostuff marshaller usage with ProtoStream marshalling.
-====
+[id='using-protostuff-marshaller_{context}']
```

Comment:
Protostuff and Kryo are deprecated, maybe we could remove it from the docs to discourage adoption? :smile:

Reply:
that sounds fine with me. I'd like to do this. @ryanemerson any reason not to drop docs for Kyro and Protostuff?